joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

wheel_left_controller:
  type: velocity_controllers/JointVelocityController
  joint: wheel_left_joint
  pid:
    p: 100.0
    i: 0.01
    d: 0.1
  publish_rate: 50

wheel_right_controller:
  type: velocity_controllers/JointVelocityController
  joint: wheel_right_joint
  pid:
    p: 100.0
    i: 0.01
    d: 0.1
  publish_rate: 50